api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java

```diff
@@ -209,4 +189,37 @@ public ParticipantCohortStatus findParticipantCohortStatus(Long cohortReviewId,
     public Slice<ParticipantCohortStatus> findParticipantCohortStatuses(Long cohortReviewId, PageRequest pageRequest) {
         return participantCohortStatusDao.findByParticipantKey_CohortReviewId(cohortReviewId, pageRequest);
     }
+
+    private void close(Connection connection) {
+        if (connection != null) {
+            try {
+                connection.close();
```

Comment:
You may want to note here that this doesn't actually close the underlying pooled connection... but is the right thing to do anyway... I think. :)